docs/sources/auth/planning/index.md

```diff
@@ -160,8 +160,22 @@ You can assign these roles to users, teams and service accounts.
 
 Moreover, RBAC allows you to create your own custom roles and edit permissions granted by Grafana's basic roles.
 
-## 🚧 Will I need synchronization?
+## User synchronization between Grafana and Identity Providers
 
-### 🚧 Team sync
+When integrating Grafana with an Identity Provider, besides the initial authentication configuration, we might want to consider user base and role synchronization. This will allow the users within a group to share the same configuration, thus preventing setting permissions for each user.
 
-### 🚧 Organization sync
+### Team sync
+
+Team sync lets you set up synchronization between your auth providers teams and teams in Grafana. This enables LDAP, OAuth, or SAML users who are members of certain teams or groups to automatically be added or removed as members of certain teams in Grafana.
+
+Grafana will keep the users in sync by actively querying for any changes. No need to worry to remove the user from accessing Grafana once he has been removed from the Identity Provider. The users will be labeled according to their Identity Provider for easy tracking.
```

Comment:
Team sync only synchronizes user's teams upon login, so technically it's not "actively querying". You could maybe say something like
```suggestion
Grafana will keep users in sync by querying for any IdP team or group changes upon user login, and adding or removing the user from teams accordingly.
```

Reply:
I also removed the last two sentences, as they are not really related to team sync. I don't think we automatically remove the users from Grafana if they've been removed from the IdP (but I need to double-check this). And I'd probably mention the labelling under `Internal vs external users` section higher above.